tests/run_tests.sh

```
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Script to run all WebCat tests
+# This script activates the virtual environment and runs the tests
+
+# Move to the project root directory
+cd "$(dirname "$0")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)